lib/ramble/ramble/cmd/license.py

```diff
@@ -65,20 +65,31 @@
 ]
 
 
-def _all_ramble_files(root=ramble.paths.prefix):
+def _get_modified_files(root):
+    """Get a list of modified files in the current repository."""
+    diff_args = ['-C', root, 'diff', 'HEAD', '--name-only']
+    files = git(*diff_args, output=str).split()
+    print('Modified files:\n', files)
```

Comment:
Do we need the print here?

Reply:
Good catch. It was a remnant from local debugging. Removed.